test/swivel/Swivel.sol

```diff
@@ -16,9 +16,10 @@ contract Swivel {
 
   string constant public NAME = 'Swivel Finance';
   string constant public VERSION = '2.0.0';
-  uint256 constant public HOLD = 259200; // obvs could be a smaller uint but packing?
+  uint256 constant public HOLD = 3 days; // obvs could be a smaller uint but packing?
```

Comment:
is this one of the .sol available constants?

Reply:
Yup, compiles fine. I'm cool w/ the change

Reply:
done in with 'gas opts'